src/styles/components/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @extend %flex-between-center;
+  &__star {
+    width: 1.6rem;
+    height: 1.6rem;
+    margin-right: 4px;
+
+    @extend %bgc-image;
+
+    background-image: url('../images/star.svg');
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &.stars--4 > .stars__star:nth-child(-n + 4) {
```

Comment:
The selector `.stars--4 > .stars__star:nth-child(-n + 4)` is quite specific. Ensure that this specificity is necessary and does not conflict with other styles. Consider using a more general class or a modifier if possible.